app/views/admin/applications/show.html.erb

```diff
@@ -0,0 +1,67 @@
+<h1><%= @application.name %></h1>
+
+<h3>Street Address: <%= @application.street_address %></h3>
+<h3>City: <%= @application.city %></h3>
+<h3>State: <%= @application.state %></h3>
+<h3>Zip Code: <%= @application.zip_code %></h3>
+<h3>Reason for applying: <%= @application.description %></h3>
+<h3>Application Status: <%= @application.status %></h3>
+
+<h3>Applying for pets: </h3>
+<ul>
+    <% @application.pets.each do |pet| %>
+        <li><p><%= link_to "#{pet.name}", "/pets/#{pet.id}" %></p></li>
+        <% if ApplicationPet.where({pet_id: pet.id,application_id: @application.id}).first.status == "Approved" %>
+            <%= "(#{pet.name} already approved)" %>
+        <% elsif ApplicationPet.where({pet_id: pet.id,application_id: @application.id}).first.status == "Rejected" %>
+            <%= "(#{pet.name} already rejected)" %>
+        <% else %>
+            <%= form_with url: "/admin/applications/#{@application.id}", method: :get, local: true do |f| %>
```

Comment:
As mentioned in controller comments, a more RESTful route for this would be "/applications/:app_id/pets/:pet_id" and this could route to "application_pets#update". Then, it would redirect to "admin/applications/:id"